airflow/providers/influxdb/hooks/influxdb.py

```diff
@@ -0,0 +1,163 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""This module allows to connect to a InfluxDB database."""
+
+from typing import List
+
+import pandas as pd
```

Comment:
mmm this provider must have `pandas` yet pandas is now an optional dependency in Airflow core https://github.com/apache/airflow/pull/17575
shouldn't we wrap it with try catch and raising exception if pandas is not installed?
@potiuk WDYT?

Reply:
Good catch. I think It's better to get influx db extra to depend on pandas directly (with 'pandas>=0.17.1, <2.0`  - Ok with that @kanthi?

Reply:
Thanks, added in setup.py